frontend/components/TableContainer/TableContainerUtils.ts

```diff
@@ -0,0 +1,9 @@
+const DEFAULT_RESULTS_NAME = 'results';
+
+const generateResultsCountText = (name: string = DEFAULT_RESULTS_NAME, pageIndex: number, pageSize: number, resultsCount: number) => {
+  if (pageSize === resultsCount) return `${pageSize}+ ${name}`;
+  if (pageIndex !== 0 && (resultsCount <= pageSize)) return `${pageSize}+ ${name}`;
+  return `${resultsCount} ${name}`;
+};
```

Comment:
Adding a case for 1 where it returns `1 result` would be nice

Reply:
for sure will do that in next PR, The hard part about that is the word is dynamic, so may be some weirdness taking off plural just for the 1 result